### ISSUE TITLE
Lab1一些疑問

### ISSUE THREAD

**Issue by jasperchang504** [2020-DL-Training-Program/Lab1-Backpropagation#2]
請問Lab中的np.kron是什麼呢
還有MSE function中derivative 這個flag要用來做什麼呢, MSE不是算一個值嗎

**Comment by zivzone** [comment]
@jasperchang504 你好，我們在講解 Lab1 時旁邊所附的 code 截圖皆為助教實作出來的 code 截圖，為參考。講解 lab 時強調產生 data 的 function 以及畫圖的 function 請使用助教提供的(可經過小幅修改，在 report 中提一下即可)，其他 function 皆可自由實作，因此不用拘限助教的 code 。np.kron 是助教在計算 gradient backward那一個 function 時所呼叫的一個 numpy 的 function，也可以不使用此 function ，採用其他的 numpy function 等等去實現。 MSE這個 function 因位助教在實作 gradient backward 時，會再利用此function，所以加了一個 flag 用以跟一般在算 predition 跟 ground truth 的 MSE loss 區隔開來。學員可以自由寫任何需要用到的 function。
